administrator/language/de-DE/de-DE.com_cpanel.ini

```diff
@@ -24,6 +24,8 @@ COM_CPANEL_MSG_EACCELERATOR_BUTTON="Auf Datei umstellen."
 COM_CPANEL_MSG_EACCELERATOR_TITLE="eAccelerator ist mit Joomla! <strong><u>nicht</u></strong> kompatibel!"
 COM_CPANEL_MSG_HTACCESS_BODY="In Joomla! 3.4 wurden Änderungen an den Standard „.htaccess“- und der „web.config“-Datei vorgenommen, die die Verzeichnisauflistung („Directory Listing“) unterbinden. Dem Websitebetreiber wird empfohlen, diese Änderung auch in seinen Dateien vorzunehmen. Weitere Informationen können in der <a href='https://docs.joomla.org/Special:MyLanguage/Preconfigured_htaccess' target='_blank' rel='noopener noreferrer'>Joomla!-Dokumentation (engl.)</a> nachgelesen werden."
 COM_CPANEL_MSG_HTACCESS_TITLE="Änderungen an der Standard .htaccess- & web.config-Datei"
+COM_CPANEL_MSG_HTACCESSSVG_TITLE="Zusätzlicher XSS-Schutz für die Verwendung von SVG-Dateien"
+COM_CPANEL_MSG_HTACCESSSVG_BODY="<p>Seit 3.9.21 wird Joomla mit einer zusätzlichen Sicherheitsregel in der Standarddatei htaccess.txt ausgeliefert. Diese Regel schützt Benutzer von svg-Dateien vor potentiellen Cross-Site-Scripting (XSS)-Schwachstellen.<br>Das Sicherheitsteam empfiehlt, die notwendigen Änderungen manuell auf jede bestehende .htaccess-Datei anzuwenden, da diese Datei nicht automatisch aktualisiert werden kann.</p><p><<stark>Änderungen für .htaccess</stark><br /></p><pre>&lt;FilesMatch \"\.svg$\"&gt;\n  &lt;IfModule mod_headers.c&gt;\n    Header always set Content-Security-Policy \"script-src 'none'\"\n  &lt;/IfModule&gt;\n&lt;/FilesMatch&gt;</pre><br>Zurzeit ist uns keine Methode zur bedingten Konfiguration auf IIS-Webservern bekannt, bitte kontaktieren Sie Ihren Hosting-Provider für weitere Unterstützung.</p>"
```

Comment:
'in der Standard htaccess.txt'

'auf die bestehende .htaccess-Datei'

'zur bedingten Konfiguration dieser Regel auf IIS-Webservern'